unity/Holo/Assets/Scripts/model_with_plate/ModelWithPlate.cs

```diff
@@ -245,16 +245,21 @@ private void ClickChangeLayerState()
 
     /* All the variables below are non-null if and only if after 
      * LoadInstance call (and before UnloadInstance). */
-    private int? instanceIndex; // index to ModelsCollection
-    private BlendShapeAnimation instanceAnimation;
-    private BlendShapeAnimation dataLayerInstanceAnimation;
+    [HideInInspector]
+    public int? instanceIndex; // index to ModelsCollection
+    [HideInInspector]
+    public BlendShapeAnimation instanceAnimation;

```

Comment:
Synchronizing layers will need to be redone after my PR https://github.com/MicroscopeIT/holo/pull/35 , but that's a problem for later )